src/enums.js

```diff
@@ -72,14 +72,17 @@ export const cropLifeStage = enumify(['SEED', 'GROWING', 'GROWN'])
  * @enum {string}
  */
 export const itemType = enumify([
+  'BRONZE_ORE',
   'COW_FEED',
   'CRAFTED_ITEM',
   'CROP',
   'FERTILIZER',
   'HUGGING_MACHINE',
   'MILK',
   'SCARECROW',
+  'SILVER_ORE',
```

Comment:
I think `BRONZE_ORE` and `SILVER_ORE` can be merged into `ORE` since there's nothing significant differentiating them aside from relative value. Did you have something else in mind when you made the two separate `itemType`s?

Reply:
oops, nope i see that now. i'm just so used to adding crops i added the individual items